src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -342,4 +343,32 @@
             </xsd:extension>
         </xsd:simpleContent>
     </xsd:complexType>
+
+    <xsd:complexType name="messenger">
+        <xsd:all>
+            <xsd:element name="routing" type="messenger_routing" minOccurs="0" maxOccurs="1" />
+            <xsd:element name="doctrine-transaction" type="messenger_doctrine_transaction" minOccurs="0" maxOccurs="1" />
+        </xsd:all>
+    </xsd:complexType>
+
+    <xsd:complexType name="messenger_routing">
+        <xsd:sequence>
+            <xsd:element name="sender" type="messenger_routing_sender" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="messenger_routing_sender">
+        <xsd:simpleContent>
+            <xsd:extension base="xsd:string">
+                <xsd:attribute name="message-class" type="xsd:string" />
+            </xsd:extension>
+        </xsd:simpleContent>
+    </xsd:complexType>
+
+    <xsd:complexType name="messenger_doctrine_transaction">
+        <xsd:sequence>
+            <xsd:element name="entity-manager-name" type="xsd:string" />
```

Comment:
this one could be an attribute rather than a child element IMO

Reply:
Could you please assist me here. I've have a hard time making this work. That is why the tests are failing.

Reply:
Last thing I guess: can you move this to be an attribute? It would mean [this XML](https://github.com/Nyholm/symfony/blob/5fa912b7d371c87e7853d1374fcdb48cf28249ec/src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/messenger_doctrine.xml) would look like that instead:
```xml
    <framework:config>
        <framework:messenger>
            <framework:doctrine-transaction entity-manager-name="foobar" />
        </framework:messenger>
    </framework:config>
```